docs/features/mysql/index.mdx

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 1
+title: MySQL | Overview
+hide_title: true
+---
+
+import DocsLinkCard from "@site/src/components/LinkCard";
+
+export const mysql_tutorials = [
+    {
+        title: 'Just-in-time MySQL Access',
+        description: 'Learn how to manage just-in-time users and SQL GRANTs',
+        url: '/features/mysql/tutorials/mysql'
+    },
+];
+
+# MySQL
+
+Otterize is able to create just-in-time username-and-password pairs for your service, providing them as a Kubernetes Secret that can be mounted to file or mapped to environment variables, as well as `GRANT`ing access to databases and tables, based on `ClientIntents` ([Intents-Based Access Control](/overview/intent-based-access-control)) declarations.
```

Comment:
```suggestion
Otterize is able to create just-in-time username-and-password pairs for your service, providing them as a Kubernetes Secret that can be mounted to file or mapped to environment variables, as well as `GRANT`ing access to databases and tables, based on `ClientIntents` ([Intent-Based Access Control](/overview/intent-based-access-control)) declarations.
```
Please also fix in PGSQL (I can't suggest changes to that one, and missed it in the original review)